action.yml

```diff
@@ -30,7 +30,7 @@ inputs:
 
 runs:
   using: docker
-  image: "docker://neondatabase/gh-workflow-stats-action:v0.2.1"
+  image: "docker://ghcr.io/neondatabase/gh-workflow-stats-action-history-exporter:v0.2.2"
```

Comment:
can you create the git tag first, so that we see that the image is actually built with this name + tag?